pages/common/home-manager.md

```diff
@@ -0,0 +1,12 @@
+# home-manager
+
+> Manage a user environment using Nix.
+> More information: <https://github.com/rycee/home-manager>.
+
+- Activate a configuration and switch to it:
+
+`home-manager switch`
```

Comment:
Here you're saying "activate a configuration and switch to it", but *which configuration*? Does this miss an argument? And if not, how does it work?

Reply:
@mebeim the config file is presumed to be located at `~/.config/nixpkgs/home.nix`.

Reply:
Maybe it would be nice to have that explicitly said on the description.

Reply:
Ah, I see. I agree that it would be better to have that stated somewhere.

Reply:
Ah, I see. I agree that it would be better to have that stated somewhere.